src/styles/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  transition: transform 0.3s ease;
+  color: $black;
+
+  &:hover {
+    transform: scale(120%);
+    color: #34568b;
+  }
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
+    background-repeat: no-repeat;
+    background-size: 160px 134px;
+    height: 166px;
+    background-position: 19px 32px;
+  }
+
+  &__apple {
+    margin: $margin;
+    padding: 40px 16px 0;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__product {
+    margin: $margin;
```

Comment:
Similar to the previous comment, the variable `$margin` is used here without being defined. Make sure it is defined or replace it with a specific value.